### DISCUSSION TITLE
[firebase_messaging] v8.0.0 action button support

### DISCUSSION THREAD

**Discussion by ghost** [firebase/flutterfire#4001] [Q&A] [answered]
Greetings,

I want to add action buttons for Android, like the picture below. 

I have been trying this almost for a week. In the first stages of my research, I understand that I have to do it with native code. Since I have no native mobile development experience, also no experience with Java or Kotlin made my problem bigger. Another crucial question in my head was code maintainability. 

So, When I found out the new version was pre-released, I immediately started to test version 8 of firebase messaging. I tried to do a few things with the code in the example project but failed. My questions:

1. Will this feature be supported in the new version?
2. Are there any code samples or documentation you can suggest? 

Thanks.

![asdf](https://user-images.githubusercontent.com/25451736/97772342-21511a00-1b57-11eb-831e-52be2e59b282.png)

**Comment by Salakar** [comment, marked as answer]
For this you will need to use a local notifications package.
